File: dayThree/python/reversed_integer.py

number =  1234;
number = str(number)
length = len(number)
number = int(number)

reverse = 0;
digit = 0;
counter  = 0


while(counter != length):
    digit = number % 10
    reverse = (reverse * 10) +  digit;
    number //= 10;
    counter +=1
    
print(reverse); 

File: dayThree/java/ReversedInteger.java
public class ReversedInteger{
    public static void main(String[] args) {
        int number =  1234;
        int reverse = 0;
        int digit = 0;
        while(number != 0) {
            digit = number % 10;
            reverse = (reverse * 10) +  digit;
            number /= 10;
        }  
        System.out.println(reverse); 
    }
}
